README.rst

```diff
@@ -11,7 +11,7 @@
 • `Contributing`_
 • `Community and Support`_
 
-|CI| |Maintainability| |Coverage| |VS Code| |DOI|
+|CI| |Python Version| |Coverage| |VS Code| |DOI|
```

Comment:
I replaced the codeclimate label. I don't think it was useful, and the readme top section is crowded with links and labels.